scipy/spatial/transform/rotation.py

```diff
@@ -129,3 +129,56 @@ def as_dcm(self):
             return dcm[0]
         else:
             return dcm
+
+    @classmethod
+    def from_dcm(cls, mat, scalar=True):
+        # For now, mat must be a 3D matrix
+        num_rotations = mat.shape[0]
+        if scalar:
```

Comment:
If we are keeping both methods, how would we organize the code?

1. Two separate self-contained functions?
2. Keep as it is written now?

I think 2 would be better. My original suggestion to factor out some logic into separate function --- I don't like it.

Reply:
Even I prefer keeping it as it is written. If we need the functionality later on in another function, then we can decide to separate out some common logic (although at this point I'm not sure if we'll require that)